README.md

```diff
@@ -10,76 +10,34 @@ Active Admin is a Ruby on Rails framework for creating elegant backends for webs
 
 ## State of the project
 
-### 1.0.0
+1.0.0.pre5 is the last release to support Rails 3.2 and Ruby 2.0 or earlier.
+It is provided to help with upgrades to Rails 4 only.
 
-We're [currently working on 1.0.0](https://github.com/activeadmin/activeadmin/issues?milestone=18),
-which as far as dependencies, moves us from meta_search to Ransack and adds Rails 4 & 5 support.
-
-You can get it by tracking master:
-```ruby
-gem 'activeadmin', git: 'https://github.com/activeadmin/activeadmin'
-```
-
-Or you can using rubygems:
-```ruby
-gem 'activeadmin', '~> 1.0.0.pre4'
-```
-
-*Keep in mind that during the time where we use `pre`-release label, things can break in each release!*
-
-### Rails 5
-
-Active Admin master has preliminary support for Rails 5. To give it a try, these Gemfile changes may be needed:
-
-```ruby
-gem 'inherited_resources', git: 'https://github.com/activeadmin/inherited_resources'
-```
```

Comment:
I think we should leave the Rails 5 instructions there until we release 1.0.0, or at least until we bump the required `inherited_resources` in the gemspec file. Not necessary if we release everything at the same time. But doing that is going to be an (unnecessary) mess in my opinion.